src/patterns/ListPreview/ListPreview.tsx

```diff
@@ -0,0 +1,243 @@
+import { ComponentProps, forwardRef, ElementType, memo, useRef } from 'react';
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import Link, { LinkProps } from '../../components/Link/Link';
+import { Text, TextVariants } from '../../components/Text';
+import { SeldonImage } from '../../components/SeldonImage';
+import { Icon } from '../../components/Icon';
+import * as Popover from '@radix-ui/react-popover';
+export interface ListPreviewProps extends ComponentProps<'div'> {
+  /**
+   * Lots display text en/zh
+   */
+  lotText?: string;
+  /**
+   * Lots display text en/zh
+   */
+  lotsText?: string;
+  /**
+   * Lot empty Favorites text en/zh
+   */
+  emptyFavoritesText?: string;
+  /**
+   * Lot blank list text en/zh
+   */
+  blankListText?: string;
+  /**
+   * Lot empty lists text en/zh
+   */
+  emptyListsText?: string;
+  /**
+   * Lot edit list text en/zh
+   */
+  editListText?: string;
+  /**
+   * Lot delete list text en/zh
+   */
+  deleteListText?: string;
+  /**
+   * List data containing count and name
+   */
+  list?: {
+    count: number;
+    name: string;
+  };
+  /**
+   * Image URL for the list
+   */
+  listImageUrl: string;
+  /**
+   * Whether this card is in favorites view
+   */
+  isFavorites?: boolean;
+  /**
+   * Whether this card is in lists view
+   */
+  isLists?: boolean;
+  /**
+   * Element to render within the navigation item, renders <Link> by default
+   */
+  element?: ElementType<LinkProps>;
+  /**
+   * imageLink
+   */
+  imageLink?: string;
+  /**
+   * Callback function for editList menu item click
+   */
+  onEditListClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  /**
+   * Callback function for deleteList menu item click
+   */
+  onDeleteListClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
why not require an `id` property for this `ListPreviewCard` and then pass it back on the `onDelete` and `onEdit` in addition to the full MouseEvent just to make it a bit easier to parse  `onDelete(id:string, event: React.MouseEvent)`.  I've seen patterns like that that I like out there.  This allows consumers to have one centralized event handlers, and also you can use the `id` property to set the `dataTestId` for testing.